www/viewgame

```diff
@@ -1359,8 +1370,200 @@ function updateUnwishList(id, stat)
+
+
+    // Check the database to see if the user has a "time_in_minutes" vote stored for this game.
+    // If one exists, return the stored time in minutes. If not, return 0.
+    $mystoredtime=0;
+
+    $result = mysqli_execute_query($db, "select time_in_minutes from playertimes where gameid = ? and userid = ?", [$id, $curuser]);
+    if (!$result) throw new Exception("Error: " . mysqli_error($db));
+        while ($row = mysqli_fetch_assoc($result)) {     
+            // There should only be one row, at most
+            $mystoredtime = $row['time_in_minutes']; 
+        }
+    // At this point $mystoredtime should reflect the player's actual stored time for this game
+    $my_time_as_text = "";
+    $time_announcement = ""; // The string shown in the Estimated Play Time control panel
+    if ($mystoredtime >= 1) {
+        $hours_and_minutes = convertToHoursAndMinutes($mystoredtime);
+        $my_time_as_text=convertTimeToText($hours_and_minutes);
+        $time_announcement = "Your vote: " . $my_time_as_text;
+    }
+   
+
+//-----------------------Estimated play time control panel begins----------------------------
+
+        echo "<div class=indented>";
+        echo "<div id='timeVoteControls' class='viewgame__playedSection'>"  // There's an event listener attached to "timeVoteControls"
+        . "<span id=\"timeVoteControlsCaption\"><b>Estimated play time:</b></span>";
+
+        echo "<div id='submitTimeSection'>"
+        . "<span id='howLong'>Vote on how long it takes to finish this game.</span><br>"
+        . "<details><summary>Tips</summary>"
+        . "<ul>"
+        . "<li>Vote only if you've played the game.</li>"
+        . "<li>Vote on how long it would take to get to one final ending without hints or a walkthrough.</li>"
+        . "<li>Time spent away from the game doesn't count.</li>"
+        . "<li>Times don't need to be exact. Good-faith estimates are fine."
+        . "<ul></details>"
+        . "<p><label for='hours'>Hours:</label> "
+        . "<input type='number' name='hours' id='hours' min=0 max=200> "
+        . "<label for='minutes'>Minutes:</label> "
+        . "<input type='number' name='minutes' id='minutes' min=0 max=59></p>"
+        . "<button id='submitTimeButton' class='fancy-button' type='submit'>Submit Time</button>"
+	    . "</div>";
+
+        echo "<div id='removeTimeSection'>"
+        . "<span id='yourTime'>" . $time_announcement . "</span><br>"
+        . "<button id='removeTimeButton' class='fancy-button'>Remove Time</button>"
+        . "</div>";
+
+        echo " <span id=\"timeSaveMsg\" class=xmlstatmsg aria-live='polite'></span><br>";
+
+	    echo "<div id='timeSavedSection'>"
+        . "<p id='timeSubmittedMessage'>Your time has been submitted.</p>"
+	    . "</div>";
+
+
+// -------------------JavaScript for the estimated play time control panel---------------------
+?>
+
+<script type="text/JavaScript" nonce="<?php global $nonce; echo $nonce; ?>">
```

Comment:
I think `type="text/JavaScript"` can be removed from all scripts

Reply:
I only removed the new one here. We can remove the others in a separate PR